sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -28,13 +28,28 @@ static StorageTestBase()
 #endif
         }
 
+        /// <summary>
+        /// Add a static TestEventListener which will redirect SDK logging
+        /// to Console.Out for easy debugging.
+        ///
+        /// This is only here to run before any of our tests make requests.
+        /// </summary>
+        private readonly TestEventListener _listener;
+
         public StorageTestBase(bool async, RecordedTestMode? mode = null)
             : base(async, mode ?? RecordedTestUtilities.GetModeFromEnvironment())
         {
+            _listener = new TestEventListener();
             Sanitizer = new StorageRecordedTestSanitizer();
             Matcher = new StorageRecordMatcher(Sanitizer);
         }
 
+        [TearDown]
+        public void CleanUp()
+        {
+            _listener.Dispose();
```

Comment:
Shouldn't the creation be in a `[Setup]` method then?  Does this currently create it once per test class and dispose it once per test method?

Reply:
I think we can use OneTimeTearDown?